.github/workflows/node-ci.yml

```diff
@@ -4,22 +4,23 @@ on: [push]
 
 jobs:
   build:
-
     runs-on: ubuntu-latest
-    
+    env:
+      USE_MOCK_DATA_CMS: 1
+      CSV_PATH: ./data/studieplekken-latest.csv
     steps:
-    - uses: actions/checkout@master
-    - uses: actions/setup-node@v1
-      with:
-        node-version: '10.x'
+      - uses: actions/checkout@v2
 
-    - name: install npm dependencies
-      run: npm install --no-optional --no-audit
-      env:
-        CI: true
+      - uses: actions/setup-node@v2
+        with:
+          node-version: "16"
```

Comment:
@Siilwyn how would we get this version from the `.nvmrc` file? I would like to have a single source of truth for the node version for local development, GitHub CI and Netlify.

Reply:
Ok, answering my own question: [Natively available in setup-node@v3](https://github.com/actions/setup-node/blob/main/docs/advanced-usage.md#node-version-file):

```yaml
steps:
- uses: actions/checkout@v3
- uses: actions/setup-node@v3
  with:
    node-version-file: '.nvmrc'
```